resources/iTerm2/readme.md

```diff
@@ -0,0 +1,29 @@
+# Remedy - Terminal
+
+> An iTerm 2 profile matching the colours used in VS Code's Remedy theme. 🎨
+
+## Installation
+
+- Download the _Remedy.itermcolors_ file from the [latest release assets][github-releases] on Github or from this directory
```

Comment:
Hey we need to add the profile to the assets. 😄 You can set that up in the _release.config.js_. It's pretty self-explanatory. 👍

Reply:
Sorry, completely missed this - never worked with semantic-release before! Fixed with cb50c20.